client/app/components/packages/pas-form/show.hbs

```diff
@@ -1,715 +1,551 @@
+          {{if (eq @package.pasForm.dcpLegalstreetfrontage 717170002) "Record Street"}}
+          {{if (eq @package.pasForm.dcpLegalstreetfrontage 717170003) "Corporation Counsel Opinion"}}
+          {{if (eq @package.pasForm.dcpLegalstreetfrontage 717170004) "Mapped but not Build"}}
+        </span>
       </p>
 
-      <p>
-        <strong>
-          Do all land use actions meet
+      <p class="grid-x ruled-adjacent tight">
+        <strong class="cell auto large-padding-right">
+          Meets
           <a href="#">
             SEQRA or CEQR criteria
           </a>
-          for Type II status?
+          for Type II status
           {{! TODO: add link }}
         </strong>
-        <br />
-        {{if (eq @package.pasForm.dcpLanduseactiontype2 "717170000") "Yes"}}
-        {{if (eq @package.pasForm.dcpLanduseactiontype2 "717170001") "No"}}
-        {{
-          if
-          (eq @package.pasForm.dcpLanduseactiontype2 "717170002")
-          "Unsure at this time"
-        }}
+        <span class="cell large-3">
+          {{if (eq @package.pasForm.dcpLanduseactiontype2 717170000) "Yes"}}
+          {{if (eq @package.pasForm.dcpLanduseactiontype2 717170001) "No"}}
+          {{if (eq @package.pasForm.dcpLanduseactiontype2 717170002) "Unsure at this time"}}
+        </span>
       </p>
 
-      {{#if (eq @package.pasForm.dcpLanduseactiontype2 "717170000")}}
-        <p>
-          <strong>
-            Indicate which SEQRA or CEQR category each action fulfills
+      {{#if (eq @package.pasForm.dcpLanduseactiontype2 717170000)}}
+        <p class="grid-x ruled-adjacent tight">
+          <strong class="cell auto large-padding-right">
+            SEQRA or CEQR category each action fulfills:
           </strong>
-          <br />
-          {{@package.pasForm.dcpPleaseexplaintypeiienvreview}}
+          <span class="cell large-3">
+            {{@package.pasForm.dcpPleaseexplaintypeiienvreview}}
+          </span>
         </p>
       {{/if}}
 
-      <p>
-        <strong>
-          Is the proposed Project Area in an
+      <p class="grid-x ruled-adjacent tight">
+        <strong class="cell auto large-padding-right">
+          In an
           <a href="#">
             Industrial Business Zone
           </a>
-          ?
           {{! TODO: add link }}
         </strong>
-        <br />
-        {{
-          if
-          (eq @package.pasForm.dcpProjectareaindustrialbusinesszone true)
-          "Yes"
-        }}
-        {{
-          if
-          (eq @package.pasForm.dcpProjectareaindustrialbusinesszone false)
-          "No"
-        }}
+        <span class="cell large-3">
+          {{if (eq @package.pasForm.dcpProjectareaindustrialbusinesszone true) "Yes"}}
+          {{if (eq @package.pasForm.dcpProjectareaindustrialbusinesszone false) "No"}}
+        </span>
       </p>
 
       {{#if (eq @package.pasForm.dcpProjectareaindustrialbusinesszone true)}}
```

Comment:
I know this is not part of the commit but IDK why we need to see if it's `true`, why not just:

```
{{#if @package.pasForm.dcpProjectareaindustrialbusinesszone}}
{{/if}}
```

Reply:
ii'd understand if we need to check if something is `false` as opposed to `null`

Reply:
I imagine this was just a result of copy/paste from one that's evaluating codes for "true."